perllib/FixMyStreet/App/Controller/Static.pm

```diff
@@ -61,6 +61,20 @@ sub council : Global : Args(0) {
     my ( $self, $c ) = @_;
 }
 
+sub unresponsive : Global : Args(0) {
+    my ( $self, $c ) = @_;
+    my $body = $c->stash->{body} = $c->model('DB::Body')->find({ id => $c->req->param('body') })
+        or $c->detach( '/page_error_404_not_found' );
+
+    my @contacts = $c->model('DB::Contact')->not_deleted->search( { body_id => $body->id } )->all;
+    my $any_unresponsive = 0;
+    foreach my $contact (@contacts) {
+        $any_unresponsive = 1 if $contact->email =~ /^REFUSED$/i;
+    }
+
+    $c->detach( '/page_error_404_not_found' ) unless $any_unresponsive;
```

Comment:
This seems to return 404 if the body's send method is `Refused` and no individual contacts are marked unresponsive.